src/main/java/org/traccar/notification/ShortMessage.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton@traccar.org)
+ * Copyright 2016 Andrey Kunitsyn (andrey@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.notification;
+
+public class ShortMessage {
```

Comment:
Do we need this new model? Now that short and long messages have the same format, I think we should be able to simplify many things.

Reply:
Fixed.

Reply:
With a new commit, I removed it and renamed the "FullMessage" to a common name "Message" and used it for formatting both full and short messages. Also, with a few other renaming to make them distinguishable.